doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,255 @@
+"""
+============================================
+Restore spotted cornea image with inpainting
+============================================
+
+Optical coherence tomography (OCT) is a non-invasive imaging technique used by
+ophthalmologists to take pictures of the back of a patient's eye [1]_.
+When performing OCT,
+dust may stick to the reference mirror of the equipment, causing dark spots to
+appear on the images. The problem is that these dark spots cover areas of
+in-vivo tissue, hence hiding data of interest. Our goal here is to restore
+(reconstruct) the hidden areas based on the pixels near their boundaries.
+
+This tutorial is adapted from an application shared by Jules Scholler [2]_.
+The images were acquired by Viacheslav Mazlin.
+
+.. [1] David Turbert, reviewed by Ninel Z Gregori, MD (2023)
+       `What Is Optical Coherence Tomography?
+       <https://www.aao.org/eye-health/treatments/what-is-optical-coherence-tomography>`_,
+       American Academy of Ophthalmology.
+.. [2] Jules Scholler (2019) "Image denoising using inpainting"
+       https://www.jscholler.com/2019-02-28-remove-dots/
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import plotly.io
+import plotly.express as px
+from scipy import sparse
+
+from skimage import (
+    color, filters, measure, morphology, restoration
+)
+from skimage.data import palisades_of_vogt
+
+
+#####################################################################
+# The dataset we are using in this example is an image sequence (a movie!) of
+# human in-vivo tissue. Specifically, it shows the *palisades of Vogt* of a
+# given cornea sample.
+
+#####################################################################
+# Load image data
+# ===============
+
+image_seq = palisades_of_vogt()
+
+print(f'number of dimensions: {image_seq.ndim}')
+print(f'shape: {image_seq.shape}')
+print(f'dtype: {image_seq.dtype}')
+
+#####################################################################
+# The dataset is an image stack with 60 frames (time points) and 2 spatial
+# dimensions. Let us visualize 10 frames towards the beginning of the
+# sequence, where we can see slight changes in illumination.
+# We take advantage of the ``animation_frame`` parameter in
+# Plotly's ``imshow`` function. As a side note, when the
+# ``binary_string`` parameter is set to ``True``, the image is
+# represented as grayscale.
+
+fig = px.imshow(
+    image_seq[12:22, :, :],
```

Comment:
```suggestion
    image_seq[::6, :, :],
```
Still 10 frames but makes the changes over time way more apparent!

Reply:
Oh, yes!! Thanks, @lagru :smiley: